src/game_api/level_api.cpp

```diff
@@ -622,6 +649,13 @@ void handle_tile_code(LevelGenSystem* self, std::uint32_t tile_code, std::uint16
         }
```

Comment:
I'm assuming that this logic here is the pre-tile-code-callback that lua scripts use, so I'm putting all of the alias stuff after this since clients would obviously want the original tile code name to be what they are editing and not some arbitrarily mapped name.